### ISSUE TITLE
Copy Paste Error on a single row

### ISSUE THREAD

**Issue by mpusarla** [handsontable/handsontable#3271]
We are using version 22.0 of HandsOnTable.

When trying to paste a single row, we are getting the following javaScript error:
`Uncaught TypeError: Cannot read property '0' of null`

Here is the jsFiddle:
https://jsfiddle.net/mpusarla/mLerhrwg/

Here is the specific use case. On create new row, we are resetting the height and invoking updateSettings. When column sorting is enabled, observeChanges is true by default. The runHookForOperation which is triggered on updateSettings when ObserveChanges is enabled, is returning a 1D array for changes. We converted it to 2D manually in the afterChange handler.
This was working good in 15.0 release. We recently upgraded from 15.0 to 22.0. In 19.0 release, HOT introduced row hiding and as part of that added afterChange event to the onPaste handler in CopyPaste plugin. This is causing the issue.

Question: We do not have requirements for row hiding and cannot upgrade to the latest version of HOT. Also, we cannot modify the source code. Is there a way for us to turn off the afterChange event that is wired in the onPaste handler outside HOT source?

I tried the following hack in our source code, but looks like skip property is ignored when the event is scheduled to runOnce. Even removing the hook didnt work as it is setting the skip property which is ignored.

```
if(source === 'paste'){
                if (instance.hasHook('afterChange')) {
                    var afterChangeEventsCount = instance.pluginHookBucket['afterChange'].length;
                    var found = false;
                    var afterChangeCallback;
                    for (var iter = afterChangeEventsCount - 1; iter >= 0; iter--)
                    {
                        afterChangeCallback = instance.pluginHookBucket['afterChange'][iter];
                        if (afterChangeCallback && afterChangeCallback.toString().indexOf('isSelColAreaCoverInputValue') >= 0)
                        {
                            found = true;
                           // Tried setting the skip flag to true  but didnt work
                          // afterChangeCallback.skip = true;
                            break;
                        }
                    }
                    if (found)
                    {
                        instance.removeHook('afterChange', afterChangeCallback);
                    }
                };
            }

```

Any help is appreciated.

Thanks,
Madhu.

**Comment by krzysztofspilka** [comment]
@mpusarla we've confirmed that it's indeed a bug on our side. Added to the pile, not sure at the moment when we can start working on it.

Thanks for reporting that!

**Comment by mpusarla** [comment]
As a temporary work around, able to get this working by adding the following snippet in the beforeChange handler:

            ```
if(source === 'paste') {
                if (instance.hasHook('afterChange')) {
                    var afterChangeEventsCount = instance.pluginHookBucket['afterChange'].length;
                    for (var iter = 0; iter < afterChangeEventsCount; iter++) {
                        var afterChangeCallback = instance.pluginHookBucket['afterChange'][iter];
                        if (afterChangeCallback && afterChangeCallback.runOnce === true &&
                        		afterChangeCallback.skip !== true &&
                                afterChangeCallback.toString().indexOf('Math.max') >= 0 &&
                                afterChangeCallback.toString().indexOf('.selectCell') >= 0) {
                            afterChangeCallback.skip = true;
                            break;
                        }
                    }
                }
        	}
```
If there is a better way to turn off a specific handler for a event, please let me know. The handlers are anonymous functions and I could'nt figure out a better way to find them.

**Comment by swistach** [comment]
This issue was fixed in 0.23.0: https://jsfiddle.net/24ak02pg/
